hs_core/management/commands/find_all_resources_with_invalid_chars.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import time
+import unicodedata
+
+from datetime import timedelta
+from django.core.management.base import BaseCommand
+from hs_core.models import BaseResource
+from pyld import jsonld
+
+logger = logging.getLogger(__name__)
+
+SCHEMA_VOCAB = {"@vocab": "https://schema.org/"}
+
+
+def _scan_ecma_string_issues(value, path="$"):
+    """
+    Recursively scan a Python object (dict/list/str/etc.) and yield
+    (path, message) for ECMA-262 JSON string incompatibilities.
+
+    Rules enforced:
+      - Control chars U+0000..U+001F are not allowed unescaped in JSON strings.
+      - The quotation mark (U+0022, ") and reverse solidus (U+005C)
+        must be escaped in JSON strings. We flag presence so callers can escape.
+      - Lone UTF-16 surrogate code points U+D800..U+DFFF are invalid Unicode.
+    """
```

Comment:
wow, this is cool!
This looks reasonable enough, it might be easiest to test it with a recent db snapshot in one of the dev environments!